superset/datasets/api.py

```diff
@@ -347,7 +347,7 @@ def delete(self, pk: int) -> Response:
     @statsd_metrics
     @rison(get_export_ids_schema)
     def export(self, **kwargs: Any) -> Response:
-        """Export dashboards
+        """Export datasets
```

Comment:
ups!

Reply:
_"Let him who has never copied-and-pasted in a rush cast the first stone"_